src/main/java/Duke.java

```diff
@@ -1,10 +1,136 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * A personal assistant chatbot.
+ */
 public class Duke {
+    private static final String INDENTED_HORIZONTAL_LINE = " ".repeat(4) + "_".repeat(60);
+    /** Array of Task objects */
+    private static final Task[] tasks = new Task[100];
+    /** Number of tasks */
+    private static int taskCount = 0;
+
+    private static String indent(String text) {
+        String[] lines = text.split(System.lineSeparator());
+        for (int i = 0; i < lines.length; i++) {
+            lines[i] = " ".repeat(5) + lines[i];
+        }
+        return String.join(System.lineSeparator(), lines);
+    }
```

Comment:
Perhaps this can be changed to `System.out.println("/t")`?

Reply:
That's true. I prefer using spaces so that I can control the amount of indentation (e.g. horizontal line indented by 4 spaces, text indented by 5 spaces). Thanks for your suggestion though!